src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,219 @@
+import { useState } from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+import { IcArrowDownLine, IcInfoCircleLine } from '@/style';
+
+import { Chip } from './Chip';
+
+const meta: Meta<typeof Chip> = {
+  title: 'Components/Chip',
+  component: Chip,
+  args: {
+    size: 'medium',
+    role: 'suggestion',
+    disabled: false,
+    selected: false,
+  },
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+      },
+      options: ['small', 'medium'],
+    },
+    role: {
+      control: {
+        type: 'radio',
+      },
+      options: ['suggestion', 'input', 'filter'],
+      description:
+        'Chip의 용법을 설정합니다. "group"은 ChipGroup 컴포넌트의 자식일때만 사용가능합니다. "input"은 selected를 true로 설정할 수 없습니다.',
+    },
+
+    selected: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    children: {
+      description: 'React.ReactNode 타입의 children을 받습니다.',
+      type: { name: 'other', value: '', required: true },
+      control: {
+        type: 'string',
+      },
+      defaultValue: '체크박스',
+    },
+  },
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+const ControlComponent = (args: object) => {
+  return (
+    <Chip size="medium" role="suggestion" {...args}>
+      Chip
+    </Chip>
+  );
+};
+
+const DefaultUsageComponent = () => {
+  return (
+    <Chip size="medium" role="suggestion">
+      sdsd
```

Comment:
여기 mdx 텍스트랑 맞춰서 칩으로 하는 거 워때요
진짜 짜잘하당............
그치만 제안하고 싶었습니다.

![image](https://github.com/user-attachments/assets/4bebd336-d123-499a-b825-e0a6d67aca1e)

```suggestion
      칩
```

Reply:
굿뜨임니다